### ISSUE TITLE
Certain masks and hats from AutoDrobe does not hide your identity

### ISSUE THREAD

**Issue by thekaio123** [space-wizards/space-station-14#10295]
The cardborg helmet, papersacks hats, joy mask, skub hat, pumpkin hat and chicken head.
All of them cover your face, but not your identity.

**Reproduction**
> Go to an AutoDrobe
> Wear any of the masks/hats cited above
> Remove your ID Card 
> See no difference in your identity while using them

**Screenshots**
![Screenshot_2](https://user-images.githubusercontent.com/61072165/182905277-91ed4f62-442a-458f-8468-2f5e21478415.png)

**Additional context**
This makes it difficult to anyone to hide their identity when needed, since few masks/hats hides your identity at the moment.
It would be nice if other masks/hats that cover your face also hides your identity, any mask that was not cited in this issue.

EDIT: Pumpkin hat was fixed, the others dont.

**IN SHORT**
Those hats/masks does not give you the disguised identity while wearing them.

**Comment by thekaio123** [comment]
Updated a bit, since this issue is 1 month old

**Comment by lzk228** [comment]
fixed, i think
